.github/workflows/build_nightly.yml

```diff
@@ -0,0 +1,152 @@
+name: build_nightly
+
+on:
+  push:
```

Comment:
this should also be removed

Reply:
Discussed that on push to develop it's fine.